src/libpmem2/x86_64/cpu.c

```diff
@@ -121,6 +121,7 @@ is_cpu_feature_present(unsigned func, unsigned reg, unsigned bit)
 	return (cpuinfo[reg] & bit) != 0;
 }
 
+#if 0
```

Comment:
can't we just remove this? If not, can you add a comment stating why it was left?

Reply:
We can, but in the future, we might need this again for something. I don't want to reinvent this code again.

Reply:
+1 (please add a comment)

Reply:
Actually, I split the change into more commits and made the second one easily revertible, if we'll need this again.